backend/app/settings.py

```diff
@@ -20,6 +20,7 @@ class Config(object):
     TEST_DIR = os.path.abspath(os.path.join(PROJECT_ROOT, "tests"))
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     SUBGRAPH_ENDPOINT = os.getenv("SUBGRAPH_ENDPOINT")
+    SUBGRAPH_RETRY_TIMEOUT = int(os.getenv("SUBGRAPH_RETRY_TIMEOUT", 10))  # seconds
```

Comment:
You can just change the name of env to SUBGRAPH_RETRY_TIMEOUT_SEC to show this is expressed by seconds. Comments are generally very likely to rotten over time and be hardly maintanable.

Reply:
+1